specs/_template/01_main.rst

```diff
@@ -0,0 +1,253 @@
+    - After this, use pip3 to install other prerequisites below:
+    - https://docs.restructuredtext.net/articles/prerequisites.html
+
+.. rst syntax quick reference
+    If you are used to markdown, then rst is going to be a bit strange at first.
+    rst is more powerful than markdown, and it is standardised.
+
+.. Where to get more info
+    - http://docutils.sourceforge.net/docs/user/rst/quickref.html
+    - http://docutils.sourceforge.net/docs/user/rst/quickstart.html
+    - http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html
+    - http://docutils.sourceforge.net/docs/ref/rst/directives.html
+
+.. <------- comments start with 2 dots
+
+.. Document Name
+    Note that the lines must be at least as long as the text
+    ====================== 
+    This is document name
+    ======================
+
+.. Headers
+    Just underline with any symbol like = or - or + or ~ or `
+    Be consistent - different symbols are taken as different header levels.
+    Examples:
+..
+    1st level header
+    ==================
+..
+    2nd level header
+    ------------------
+..
+    3rd level header
+    ~~~~~~~~~~~~~~~~~~~
+..
+    4th level header
+    ++++++++++++++++++
+
+.. Lists
+    1. first item
+    2. second item
+    #. auto-generated item number
+
+.. Unordered lists
+    * first level
+    <blank line>
+        - second level
+    <blank line>
+        + third level
+
+.. Code
+    .. code:: <language (optional)>
+    <blank line>
+        this is going to be code
+
+.. Links
+    `This is link text`_ <-------- undersocre at the end
+    .. _`This is  link text`: https://example.org <------------- this is link definition
+    .. anchor: <---------- this is anchor to the text below it
+    `anchor`_ <----------- this is link to the anchor
+    `Some text <anchor_>`_ <----- text is different from the anchor name
+    We put all link definitions (except from the page info header table) to the end of the document.
+
+.. This is a document header
+    It links to the epic ticket number, and provides document metadata
+
+.. Template information:
+    SPECIFICATION_NAME = name of the specification (title)
+    EPIC = epic issue number
+    AUTHOR = your name
+    CREATED_AT = YYYY-MM-dd
+
+.. Please replace the strings below with the values that you want to put in in all of the rst files of the specification.
+
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
+
+.. _Main:
+
+==================
+SPECIFICATION_NAME
+==================
+
+1. `Main`_
+2. User Interface
+
+   1. `Android`_
+   2. `Chrome`_
+   3. `iOS`_
+
+3. `External Communication`_
+4. Other_
+
+
+.. contents:: Table of Contents
+
+1. Problem Definition
```

Comment:
I would not number the sections specifically. Please use `.. sectnum::` for this.